### PR TITLE
Feat/aias5.3

### PR DESCRIPTION
**Pull request by sylvaingaudan**
Add aias with enrich and download

### CODE REVIEW

File: conf/aias.env [hunk 1]
```diff
@@ -21,12 +21,15 @@ AIRS_INDEX_COLLECTION_PREFIX=org.com@airs
...
 #APROC_RESOURCE_ID_HASH_STARTS_AT=
 
+DOWNLOAD_S3_BUCKET=downloads
 APROC_DOWNLOAD_DIR=/tmp/outbox
 APROC_INPUT_STORAGE_TYPE="https"
 APROC_INPUT_STORAGE_BUCKET=""
 APROC_INPUT_STORAGE_API_KEY_PROJECT=""
 APROC_INPUT_STORAGE_API_KEY_PRIVATE_KEY_ID=""
 APROC_INPUT_STORAGE_API_KEY_PRIVATE_KEY=""
+APROC_INPUT_STORAGE_DOMAIN=geodes-portal.cnes.fr
```
Comment (QuCMGisaia) [inline conf/aias.env:31]: Do we leave that there ?

File: conf/arlas.env [hunk 2]
```diff
@@ -100,3 +100,14 @@ ARLAS_HUB_LINKS='
...
+ARLAS_DOWNLOAD_PROCESS_URL=/aproc/processes/download/execution
+ARLAS_DOWNLOAD_PROCESS_CHECK_URL=/aproc/processes/download
+#ARLAS_DOWNLOAD_PROCESS_MAX_ITEMS
+#ARLAS_DOWNLOAD_PROCESS_SETTINGS_URL
+ARLAS_DOWNLOAD_PROCESS_STATUS_URL=/aproc/jobs
+ARLAS_ENRICH_PROCESS_URL=/aproc/processes/enrich/execution
+ARLAS_ENRICH_PROCESS_CHECK_URL=/aproc/processes/enrich
+#ARLAS_ENRICH_PROCESS_MAX_ITEMS
+#ARLAS_ENRICH_PROCESS_SETTINGS_URL
+ARLAS_ENRICH_PROCESS_STATUS_URL=/aproc/jobs
```
Comment (QuCMGisaia) [inline conf/arlas.env:113]: Missing the empty line at the end

**Review by QuCMGisaia — APPROVED** [review]

File: conf/aias.env (hunk 1 above)
Comment (sylvaingaudan) [reply conf/aias.env:31]: yes

File: README.md [hunk 3]
```diff
@@ -119,13 +119,17 @@ You can access ARLAS just like the IAM deployement. You can also use the same sc
 
 To setup an ARLAS EO catalog:
 1) place some geotiff files in ${APROC_INPUT_DIR} configured in [conf/aias.env](conf/aias.env). 
-2) Go to the web page of ARLAS, then click the **Import** link (top right menu). This brings you to the import page. Add the geotiff file in the catalog by clicking on the + icon next to the tiff file.
+2) Go to the web page of ARLAS, then click the **Import** link (top right menu). This brings you to the import page. Add the geotiff file in the catalog by clicking on the + icon next to the tiff file. This will create an automatically feed an index named "org.com@airs_catalog"
```
Comment (MohamedHamouGisaia) [inline README.md]: **and** automatically ?

**Review by MohamedHamouGisaia — APPROVED** [review]

**Pull request merged**